api/api.proto

```diff
@@ -514,14 +516,29 @@ message SnapCreateResponse {
   VolumeCreateResponse volume_create_response = 1;
 }
 
-// VolumeInfo 
+// VolumeInfo
 // swagger:model
 message VolumeInfo {
   string volume_id = 1;
   string path = 2;
   VolumeSpec storage = 3;
 }
 
+// VolumeConsumer
+// swagger: model
+message VolumeConsumer {
+  // Name is the name of the volume consumer
+  string name = 1;
+  // Namespace is the namespace of the volume consumer
+  string namespace = 2;
+  // Node is the node on which the consumer is running
+  string node = 3;
```

Comment:
Can we rename it to node_id or node_ip depending on what it is?

Reply:
This will be according to the Container Orchestration or Container Runtime. Renamed to NodeID and added a more detailed comment in incremental.